src/client/src/containers/PrivacyModal/index.tsx

```diff
@@ -79,20 +85,22 @@ const PrivacyModal = (props: Props) => {
       </div>
       <div className={styles.section}>
         {intl.formatMessage(messages.thirdPartyWebsite, {
-          thirdPartyWebsite: downloadLinkLabel
+          thirdPartyWebsite: redirectLinkLabel
         })}
       </div>
       <div className={styles.footerContainer}>
+        {privacyStatementLink && (
+          <a
+            target={"_blank"}
```

Comment:
NIT: Anchor tags with target="_blank" should also include rel="noreferrer noopener"tslint(react-anchor-blank-noopener)